tests/garage/tf/algos/test_ddpg.py

```diff
@@ -103,7 +102,8 @@ def test_ddpg_pendulum_with_decayed_weights(self):
         This environment has a [-3, 3] action_space bound.
         """
         with LocalTFRunner(snapshot_config, sess=self.sess) as runner:
-            env = normalize(GymEnv('InvertedPendulum-v2'))
+            env = normalize(
+                GymEnv('InvertedPendulum-v2', max_episode_length=100))
```

Comment:
Same for `InvertedPendulum-v2`.`env.spec. max_episode_length ` returns 1000, should we override it with 100? or 200? Plus, `max_episode_length=1000` look wrong for `InvertedPendulum-v2`... @ahtsan